security-demo/release/app-agent-conf/agent/dummy_root_ca.crt

```diff
@@ -0,0 +1,12 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Please remove the cert file in source codes.

Reply:
These are demo certificates which will ease the usage of example. The certificates are taken from spire releases. Goal is to keep user agnostic of generation certificates for the demo.

Alternate way is to create another script for generating these certificates.

Let me know your suggestions based on the above rationale.

Reply:
Let's generate the certificates - it's sensitive to upload the cert files alone with source codes.